uaa/src/test/java/org/cloudfoundry/identity/uaa/orchestrator/OrchestratorClaimAT.java

```diff
@@ -1,124 +1,265 @@
-import com.ge.predix.iam.kubernetes.utils.OrchestratorUtil;
+import java.util.Random;
+import static org.junit.Assert.*;
 
 @RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration(classes = OrchestratorIntegrationTestConfig.class)
+@Slf4j
 public class OrchestratorClaimAT {
+    protected static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
 
-    @Value("${kubernetes.config.secondary-namespace}")
-    private String namespace;
+    private static final Logger LOGGER = LoggerFactory.getLogger(OrchestratorClaimAT.class);
+
+    private static final String UAA_ADMIN_CLIENT_SECRET = "ZHVtbXlBZG1pblNlY3JldDEyMyMh";
+
+    private static final String UAA_CLAIM_NAME_PREFIX = "uaa-it-";
+
+    public static final String ZONES_URI = "/orchestrator/zones";
+
+    private static HttpHeaders headers;
+
+    private Random random = new Random();
+
+    private final String UAA_SUBDOMAIN = "rest-uaa-subdomain" + random.nextInt(1000);
+
+    private UaaRequest uaaRequest;
+
+    @Value("${uaa.url}")
+    private String uaaUrl;
+
+    @Value("${uaa.orchestrator.clientId}")
+    private String clientId;
+
+    @Value("${uaa.orchestrator.clientSecret}")
+    private String clientSecret;
+
+    @Value("${kubernetes.config.env}")
+    private String environment;
 
     @Autowired
     private OrchestratorUtil orchestratorUtil;
 
-    @Autowired
-    private AssumeRoleUtil assumeRoleUtil;
+    protected static OAuth2RestTemplate trustedIssuerValidScopesRestTemplate;
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(OrchestratorClaimAT.class);
-    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    @Value("${kubernetes.config.secondary-namespace:iamqa-svc}")
+    private String namespace;
 
-    @BeforeClass
-    public static void setupClassLevelData() {
-        LOGGER.info("In OrchestratorClaimAT.setupClassLevelData()");
-    }
+    private Integer uaaInstanceCount = random.nextInt(1000);
 
     @Before
-    public void setupTestLevelData() {
-        LOGGER.info("In OrchestratorClaimAT.setupTestLevelData()");
+    public void setupDataBeforeEachTestRun() {
+        uaaRequest = getUaaRequest();
     }
-
+    // region Positive Tests
     @Test
-    public void dummyTestTryAwsLogin() throws Exception {
-        LOGGER.info("In OrchestratorClaimAT.dummyTestTryAwsLogin()");
-        LOGGER.info("Print loaded env vars and autowired beans");
-        LOGGER.info("AssumeRoleUtil: {}", assumeRoleUtil);
-        LOGGER.info("OrchestratorUtil: {}", orchestratorUtil);
-        LOGGER.info("Loaded env var kubernetes.config.secondary-namespace: {}", namespace);
-
-        LOGGER.info("\nPerform gossamer aws login and assume role\n");
-        LOGGER.info("\nK8S Api Client returned: {}", assumeRoleUtil.getKubernetesApiClient());
-    }
+    public void testCreateZone() throws Exception {
+        String claimName;
+        OrchestratorZoneResponse getZone = null;
+        String zoneName = null;
+        claimName =
```

Comment:
Combine with variable declaration in line#86 into a single line:
`String claimName = (UAA_CLAIM_NAME_PREFIX + (uaaInstanceCount)).toLowerCase();`

Reply:
Done